### PR TITLE
Dayjs localization updates

### PR DESCRIPTION
**Pull request by tnekret**
#### Description:

Matched dayjs locales to app locales.

This is work in progress, waiting on translations.

### CODE REVIEW

File: app/constants/locale.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+export const CONVERT_ISO_TO_IETF = {
```
Comment (johnschoeman) [inline app/constants/locale.js]: if this is only being used in the languages.js file, what do you think about inlining it to that file?

**Review by johnschoeman — APPROVED** [review]

**Review by tparesi — APPROVED** [review]
LGTM

File: app/constants/locale.js (hunk 1 above)
Comment (tnekret) [reply app/constants/locale.js]: fixed!

**Comment by summetj** [comment]
Should this be stale?

**Review by JacobJaffe — APPROVED** [review]
🔥 this is elegant

**Pull request merged**